docs/01_cva6_user/CVX_Interface_Coprocessor.rst

```diff
@@ -23,12 +23,249 @@ CV-X-IF Interface and Coprocessor
 
 CV-X-IF interface specification
 -------------------------------
-Need to step1 verification. TSS/Guillaume will do.
-Refer to the CV-X-IF specification, mention the 3 supported protocol interfaces, identify the CVA6 specific features.
+
+Description
+~~~~~~~~~~~
+This design specification presents global functionalities of

```

Comment:
This is the user guide, not the design document. It should focus on the CVA6 user: integrators, SW developers... We should not find "inner" information about CVA6.